apps/site/pages/en/about/previous-releases.mdx

```diff
@@ -14,12 +14,28 @@ Production applications should only use _Active LTS_ or _Maintenance LTS_ releas
 
 ![Releases](https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true)
 
-Full details regarding Node.js release schedule are available [on GitHub](https://github.com/nodejs/release#release-schedule).
+Full details regarding the Node.js release schedule are available [on GitHub](https://github.com/nodejs/release#release-schedule).
 
 ### Commercial Support
 
-Commercial support for versions past Maintenance phase is available through our OpenJS Ecosystem Sustainability Program partner [HeroDevs](https://herodevs.com/).
+Commercial support for versions past the Maintenance phase is available through our OpenJS Ecosystem Sustainability Program partner [HeroDevs](https://herodevs.com/).
 
-## Looking for latest release of a version branch?
+## Looking for the latest release of a version branch?
 
 <DownloadReleasesTable />
+
+## Official versus Community
+
+The Node.js website offers numerous installation methods that allow Node.js to be installed in a non-interactive manner,
+for example, via CLIs, OS package managers (such as `apt`), or Node.js version managers (such as `nvm`).
+
+The Node.js project, in an attempt to popularize and advertise community efforts, has introduced a new Downloads page that lists both Official and Community installation methods, providing more versatility and options for users.
+With this change, we introduced the concept of "Official" and "Community" installation methods, which have the following key differences:
+
+|                                                                                                        | Official | Community |
+| ------------------------------------------------------------------------------------------------------ | -------- | --------- |
+| New Node.js releases are available simultaneously when Node.js releases come out                       | ✅       | ❌        |
+| Project maintainers have a close relationship with Node.js, at least a direct communication way        | ✅       | ❌        |
+| Installation method downloads the official binaries bundled by the Node.js project                     | ✅       | ❌        |
+| Installation method builds from source and/or modifies what gets bundled (SSL version, Corepack, etc.) | ❌       | ✅        |
```

Comment:
I feel this is going to cause more confusion -- as written, it seems to imply all community methods will build from source only?

Reply:
I mean, yes, that is exactly what will make an unofficial (community) method versus an official method.

It is not implying, it is what determines what official vs community is.

Reply:
Hence any community method that checks the official pre-requisite can become an official one.

Or maybe instead of calling official we just call'em "Recommended"

Reply:
My point is that a "community" method may fail on one point here (I suspect most likely the direct contact point), but this table (to me) would then imply that the method fails on all the points (is building from source and has delays for new releases).

Reply:
I understand your point. The idea is that official methods must check all of these items. Feel free to suggest a patch or change; otherwise, it is not helpful.

Reply:
I'm on mobile so don't have a great ability to submit a direct suggested change -- I think what might be better though is to just have bullet points of the qualifications for "official", and not specify anything for "community".

Reply:
That is helpful. Thanks! Im just also having issues on how to visualise this differently 😅 but that makes hella sense!

Reply:
I pushed a commit.